komga-webui/src/components/dialogs/EditSeriesDialog.vue

```diff
@@ -374,6 +412,10 @@ export default Vue.extend({
       },
       genresAvailable: [] as string[],
       tagsAvailable: [] as string[],
+      selectedThumbnail: '',
+      uploadQueue: [] as File[],
+      deleteQueue: [] as SeriesThumbnailDto[],
```

Comment:
probably worth prefixing those variables with thumbnails or poster, given we are in a larger context of editing the whole series